extensions/llama/python/llama.py

```diff
@@ -83,9 +87,15 @@ async def _run_inference_helper(self, model_input, options) -> List[Output]:
                 if options:
                     options.stream_callback(data, acc, index)
             print(flush=True)
+
+            output_data_value : str = ''
+            if isinstance(acc, str):
+                output_data_value = acc
+            else:
+                raise ValueError(f"Output {acc} needs to be of type 'str' but is of type: {type(acc)}")
```

Comment:
Is this actually needed? Can't we just type `acc: str = ""` above?

Reply:
It shouldn't ever happen, but it is possible that a non-string value gets passed to the streamer. I guess we have control of our own model parser where we pass in the streamer, so this should be fine, more just for being safe